lm_eval/tasks/mmlu/default/_default_template_yaml

```diff
@@ -15,3 +15,4 @@ metric_list:
   - metric: acc_norm
     aggregation: mean
     higher_is_better: true
+num_fewshot: 5
```

Comment:
I'm going to change this one to 0-shot default if that's alright?

Reply:
Okay, I've reverted back.